src/components/LoginOptions/LoginOptions.stories.js

```diff
@@ -76,18 +77,26 @@ export const Display = {
         url: '#',
       },
     ],
+    cosignLoginOptions: [
```

Comment:
Since this could be multiple options, maybe we could add one more to see the behavior?